WalletWasabi.Gui/Controls/ExtendedTextBox.cs

```diff
@@ -20,6 +21,16 @@ public class ExtendedTextBox : TextBox, IStyleable
 	{
 		private MenuItem _pasteItem = null;
 
+		private Subject<string> _textPasted = new Subject<string>();
+
+		public IObservable<string> TextPasted
```

Comment:
=> _textPasted?.AsObservable() ?? "";

Reply:
Cannot fix this with error handling - it should be thrown exception as this is a fundamental coder mistake if `_textPasted` is null.